models/classes/runner/QtiRunnerService.php

```diff
@@ -1343,18 +1343,19 @@ public function storeTraceVariable(RunnerServiceContext $context, $itemUri, $var
 
     /**
      * Starts the timer for the current item in the TestSession
+     *
      * @param RunnerServiceContext $context
+     * @param $timestamp
```

Comment:
Could you please explain what means the timestamp is used for, and when it's null